pkg/operator/ceph/object/spec.go

```diff
@@ -104,6 +105,34 @@ func (c *clusterConfig) makeRGWPodSpec(rgwConfig *rgwConfig) v1.PodTemplateSpec
 	return podTemplateSpec
 }
 
+func (c *clusterConfig) setPodPlacement(pod *v1.PodSpec, p rook.Placement) {
+	p.ApplyToPodSpec(pod)
+
+	// label selector for gateways used in anti-affinity rules
+	podAntiAffinity := v1.PodAffinityTerm{
+		LabelSelector: &metav1.LabelSelector{
+			MatchLabels: map[string]string{
+				k8sutil.AppAttr: AppName,
+				"rgw":           c.store.Name,
```

Comment:
I think you can use `c.getLabels()` here instead of "hard-coding" the rgw key.
Any particular reasons not to use it?